blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+---
+title: “Blog- Team Serv0101”
+layout: post
+created_at: Wed Sept 13 2017
+permalink: blog/2017-09-13-final-blog-post-team-Serv0101
+current: blog
+author: Rakhi Sharma And Neha Yadav
+twitter: @atbrakhi
+categories:
+- student-posts
+---
+
+## Blog- Team Serv0101
+
+We have been working on the Servo project, Working on this project has broadened our programming knowledge especially in Rust. We have interacted with Git lot. At this point, we cannot say that we are masters with Rust, but we are much better than when we started learning it.
+We just can’t believe that RGSoC is coming to an end, time flies. 
+
+### Learning and Challenges
+
+We were absolutely unfamiliar with Rust, and it has quite a bit of a learning curve. We struggled with Rust.  You need a lot of patience when you are learning something new it really takes a lot of time. We faced a lot of problems while understanding the codebase, specification, building environment etc. But yeah, Kudos to our coaches Rahul, Ravi and our Mentor Josh they were always up for help that was the only thing helped us in keep going. We both believe, we have learned a lot of things during last two months and still learning.
```

Comment:
Remove extra space before a sentence "You need a ...".
Break the sentence and do the changes as follows:

"But yeah, Kudos to our coaches Rahul, Ravi and our Mentor Josh **who** were always up for **the** help. That was the ..."

Change 'learned a lot of things' -> 'learned lot of things'